lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -115,13 +118,29 @@ module.exports = function(self, options) {
             } else {
               info.portrait = true;
             }
+
+            if (info.extension === 'jpg' || info.extension === 'jpeg') {
+              const data = exif.parseSync(file.path);
```

Comment:
what if data is null? Makes sense not to crash then.

Reply:
Null data will come back as an empty object, so I don't think we'll crash here.